int32 num_agents
int32 num_goals

float64[] start_x
float64[] start_y
float64[] start_z
float64[] start_theta

float64[] goal_x
float64[] goal_y
float64[] goal_z
float64[] goal_theta

float64 eps_comm
